target/linux/ath79/image/generic.mk

```diff
@@ -50,6 +50,25 @@ define Build/cybertan-trx
 	-rm $@-empty.bin
 endef
 
+define Build/edimax-headers
+	$(eval edimax_magic=$(word 1,$(1)))
+	$(eval edimax_model=$(word 2,$(1)))
+
+	$(STAGING_DIR_HOST)/bin/edimax_fw_header -M $(edimax_magic) -m $(edimax_model)\
+		-v $(VERSION_DIST)$(firstword $(subst +, , $(firstword $(subst -, ,$(REVISION))))) \
+		-n "uImage" \
+		-i $(KDIR)/loader-$(DEVICE_NAME).uImage \
+		-o $@.new;
+	$(STAGING_DIR_HOST)/bin/edimax_fw_header -M $(edimax_magic) -m $(edimax_model)\
+		-v $(VERSION_DIST)$(firstword $(subst +, , $(firstword $(subst -, ,$(REVISION))))) \
+		-n "rootfs" \
+		-i $@ \
+		-o $@.rootfs;
```

Comment:
Technically, these semicola are not needed here; they would only be needed if you chained the commands by using `\` for the final line as well. Personally, I would not make this more complicated for now by any sophisticated chaining of commands, and just remove both semicola.

Reply:
OK, done